### PR TITLE
WIP: Create readme

### PR DESCRIPTION
**Pull request by filipblnt**
see https://github.com/blindnet-io/devrel-management/issues/9 & https://github.com/blindnet-io/openness-framework/pull/33

### CODE REVIEW

**Comment by noelmace** [comment]
https://github.com/orgs/blindnet-io/teams/engineering: you should still add the installation steps

API Ref can be addressed after this PR, but is a top priority.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  <a href="https://github.com/blindnet-io/{project-short-name}/issues">Submit an Issue</a>
+  &nbsp;•&nbsp;
+  <a href="https://join.slack.com/t/blindnet/shared_invite/zt-1arqlhqt3-A8dPYXLbrnqz1ZKsz6ItOg">Online Chat</a>
+  <br>
+  <br>
+</p>
+
+## About
+
+This is the .Net Core [server-side SDK](https://docs.blindnet.io/docs/extra/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:
```
Comment (m4rk055) [inline README.md:29]:
```suggestion
This is the .Net Core [server-side SDK](https://docs.blindnet.io/docs/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:
```

Comment (m4rk055) [reply README.md:29]: Is this line copied from somewhere? The bad URL appeared in almost every PR

Comment (noelmace) [reply README.md:29]: I reused https://github.com/blindnet-io/sdk-node-server/pull/2/commits/4fefd834a93d84d9194ac2661a6fc0658891d88e, and copy-pasted the resulting README template for all other server SDK. Sorry for that.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+:rocket: Check out our [Quick Start Guide](https://blindnet.dev/docs/quickstart) to get started in a snap.
+
+## Installation
+
+<!-- FIXME: add install steps -->
+Use [{package manager or plateform}][install-tool] to install {project's name}:
+
+```bash
+{install-command}
+```
```
Comment (noelmace) [inline README.md:45]: TODO @blindnet-io/engineering

File: README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+<!-- FIXME: add API Reference
+## Usage
+📑 The API reference of blindnet devkit Node.js Server SDK is available on [blindnet.dev](https://docs.blindnet.io/docs/api_reference/server/node.js/latest).
+-->
```
Comment (noelmace) [inline README.md:50]: TODO @blindnet-io/engineering

**Review by noelmace — COMMENTED** [review]
TODO List for @blindnet-io/engineering before merging this in order to close https://github.com/blindnet-io/devrel-management/issues/9

**Comment by noelmace** [comment]
@jovana-todorovic you should NOT have merged this PR! 3 comments weren't addressed and this goes against all our processes and rules as you weren't an assignee and it had no approving reviews. Please create another PR to address the remaining comments right away.

**Comment by jovana-todorovic** [comment]
hey, the actual code implementation that should be in the library is not on this branch i needed to merge in order to pack and publish to NuGet store. Someone made a mess with this branches, license on one branch, readme on other and source code on the third. I wasn't able to deploy to NuGet store without that.

**Pull request merged**